### PR TITLE
added sorting optional for location

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

File: addons/default/visiosoft/location-module/resources/config/settings/settings.php [hunk 1]
```diff
@@ -88,4 +88,24 @@
...
         ],
     ],
+    'sorting_column' => [
+        'type' => 'anomaly.field_type.select',
+        'config' => [
+            'default_value' => 'slug',
+            'options' => [
+                'slug' => 'slug',
+                'id' => 'id',
+                'order' => 'order'
```
Comment (diashalabi) [inline addons/default/visiosoft/location-module/resources/config/settings/settings.php:98]: trans?

File: addons/default/visiosoft/location-module/src/Http/Controller/AjaxController.php [hunk 2]
```diff
@@ -101,7 +121,14 @@ public function getVillage()
...
             $id = explode(',', $this->request->id);
-            return $this->village_model->whereIn('parent_neighborhood_id', $id)->orderBy('order', 'ASC')->get();
+
+            $query = $this->village_model->whereIn('parent_neighborhood_id', $id);
+
+            $sorting_type = setting_value('visiosoft.module.location::sorting_type');
+            $sorting_column = setting_value('visiosoft.module.location::sorting_column');
+
+            return $query->orderBy($sorting_column, $sorting_type)->get();
+
```
Comment (diashalabi) [inline addons/default/visiosoft/location-module/src/Http/Controller/AjaxController.php:131]: DRY

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/location-module/resources/config/settings/settings.php (hunk 1 above)
Comment (spektra2147) [reply addons/default/visiosoft/location-module/resources/config/settings/settings.php:98]: These are the column names

File: addons/default/visiosoft/location-module/src/Http/Controller/AjaxController.php (hunk 2 above)
Comment (spektra2147) [reply addons/default/visiosoft/location-module/src/Http/Controller/AjaxController.php:131]: OK. I can do a function.

**Review by diashalabi — APPROVED** [review]

**Review by muammertop — APPROVED** [review]

**Pull request merged**